network/network_utils.py

```diff
@@ -0,0 +1,93 @@
+from scapy.all import *
+from scapy.layers.l2 import Ether, ARP
+
+import netifaces
+
+ARP_REQUEST = 1
+ARP_REPLY = 2
+ARP_REQUEST_TIMEOUT = 5  # seconds
+IP_PART_SEPARATOR = "."
+BROADCAST_MAC = "ff:ff:ff:ff:ff:ff"
+
+
+def get_mac(target_ip, mac_resolve_max_tries=5):
+    """
+    queries network for MAC address corresponding to :param target_ip
+    :param target_ip: IP address
+    :param mac_resolve_max_tries: amount of times to try resolve mac address
+    :return: mac address that matches :param target_ip
+    """
+    arp_response = None
+    try_count = 0
+
+    while not arp_response:
+        if try_count > mac_resolve_max_tries:
+            raise ValueError("cannot resolve {}. are you sure it exists ?".format(target_ip))
+
+        arp_request = Ether(dst=BROADCAST_MAC) / ARP(op=ARP_REQUEST, pdst=target_ip)
+        arp_response = srp(arp_request, timeout=ARP_REQUEST_TIMEOUT, verbose=False)[0]
+        try_count += 1
+
+    target_mac = arp_response[0][0][1].hwsrc
+    return target_mac
+
+
+def get_default_gateway_ip():
+    data = netifaces.gateways()
+    return list(data['default'].values())[0][0]
```

Comment:
Apparently AF_INET is also defined in netifaces.AF_INET already as a constant 2. Use that